cmd/cmd.go

```diff
@@ -78,6 +78,20 @@ func addDatabase(c *cobra.Command) *cobra.Command {
 	return c
 }
 
+var extraIndexesCmd = &cobra.Command{
+	Use:   "extra-indexes index-1 [index-2 …]",
```

Comment:
O principal é sinalizar que `index1` é um argumento obrigatório (remover os `-` é perfumaria, mas simplifica)

```suggestion
	Use:   "extra-indexes <index1> [index-2 …]",
```